src/main/java/mate/academy/controller/AuthenticationController.java

```diff
@@ -0,0 +1,25 @@
+package mate.academy.controller;
+
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import mate.academy.dto.user.UserRegistrationRequestDto;
+import mate.academy.dto.user.UserResponseDto;
+import mate.academy.exception.RegistrationException;
+import mate.academy.service.UserService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/auth")
```

Comment:
```suggestion
@RequestMapping("/auth")
```

Add `server.servlet.context-path=/api` in application.properties